llvm/lib/Target/AArch64/AArch64Subtarget.cpp

```diff
@@ -77,14 +77,19 @@ static cl::opt<AArch64PAuth::AuthCheckMethod>
                                         "to authenticated LR during tail call"),
                                cl::values(AUTH_CHECK_METHOD_CL_VALUES_LR));
 
+static cl::opt<unsigned> AArch64MinimumJumpTableEntries(
```

Comment:
Do we still need another option for AArch64 here as we already have `-min-jump-table-entries` in `TargetLoweringBase.cpp`？

Reply:
OK, I get it. Sorry for the noise.